### PR TITLE
Bug 926233 - try server support

### PR DESCRIPTION
**Pull request by lightsofapollo**

### CODE REVIEW

File: lib/ftp_filter.js [hunk 1]
```diff
@@ -17,7 +18,12 @@ function FTPFilter(options) {
 
 FTPFilter.prototype = {
   locate: function(path, filter, callback) {
+    debug('ftp', 'ls', path)
     this.ftp.ls(path, function(err, list) {
+      if (process.env.DEBUG) {
```
Comment (jugglinmike) [inline lib/ftp_filter.js:23]: I'm still new to the `debug` module, but I think this logic is baked-in.

File: lib/channels/try.js [hunk 2]
```diff
@@ -0,0 +1,49 @@
+var fsPath = require('path'),
+    urls = require('../urls'),
+    handleOpts = require('../options'),
```
Comment (jugglinmike) [inline lib/channels/try.js]: This variable is unused

File: lib/channels/try.js [hunk 3]
```diff
@@ -0,0 +1,49 @@
+var fsPath = require('path'),
```
Comment (jugglinmike) [inline lib/channels/try.js]: This variable is unused

File: lib/ftp_filter.js (hunk 1 above)
Comment (lightsofapollo) [inline lib/ftp_filter.js:23]: I will make a note of this in the code but I check manually so the map call is not run in production

**Pull request merged**